src/Commands/index.ts

```diff
@@ -39,9 +55,25 @@ const commandMap: {
 export const registerCommands = async (client: Client) => {
   const rest = new REST({ version: '9' }).setToken(token);
   const builtCommands = Object.keys(commandMap).map((key) => {
-    return new SlashCommandBuilder()
+    const config = commandMap[key];
+    const command = new SlashCommandBuilder()
       .setName(key)
-      .setDescription(commandMap[key].description);
+      .setDescription(config.description);
+
+    if (config.options && config.options.length) {
```

Comment:
Im starting to think that this refactor is not the best way moving forward. With just commands and descriptions this is great but with the need for params its starting to get a bit convoluted. Thoughts?